frameworks/elastic/docs/limitations.md

```diff
@@ -10,7 +10,7 @@ The maximum number of deployable nodes is constrained by the DC/OS cluster's res
 
 ## Upgrades and rolling configuration updates do not wait for green status
 
-The `serial` deploy strategy does wait for the cluster to reach green before proceeding to the next node.
+During deployment and upgrades, the `serial` strategy does not wait for the cluster to reach green before proceeding to the next node.
```

Comment:
Can we be specific and say `Elastic` service? `cluster` is a pretty overloaded term.

Reply:
done